packages/vue-components/src/Searchbar.vue

```diff
@@ -267,6 +267,13 @@ export default {
         max-width: 25.4em; /* twice of min-width, to accommodate a range of lengths */
     }
 
+    @media screen and (max-width: 768px) {
+      .form-control {
+          min-width: 8em;
+          max-width: 16em; /* twice of min-width, to accommodate a range of lengths */
+        }
```

Comment:
Some styling issue here causing the linting to fail. You may want to run `npm run lintfix` to help lint the code next time :)

Reply:
Yes, that's right! Got to make running `npm run lint` a habit. Didn't realise the styles are using 4 spaces instead of 2. Thanks for the advice!!